migration/migration3_test.go

```diff
@@ -0,0 +1,23 @@
+package migration
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/textileio/powergate/tests"
+)
+
+func TestV3(t *testing.T) {
+	t.Parallel()
+
+	ds := tests.NewTxMapDatastore()
+
+	pre(t, ds, "testdata/v3_StorageJobs.pre")
+	txn, _ := ds.NewTransaction(false)
+
+	err := V3StorageJobsIndexMigration.Run(txn)
+	require.NoError(t, err)
+	require.NoError(t, txn.Commit())
+
+	post(t, ds, "testdata/v3_StorageJobs.post")
```

Comment:
Not much to test here other than the data is what we expect according to the csv files.